__tests__/.eslintrc

```
@@ -0,0 +1,3 @@
+{
+  "extends": ["@readme/eslint-config/testing/jest", "@readme/eslint-config/testing/vitest"]
```

last i remember those two configs didn't always play well with each other so we can probably just run the vitest one?
 
```suggestion
  "extends": ["@readme/eslint-config/testing/vitest"]
```

---

oh oops lol: https://github.com/readmeio/jest-expect-har/commit/9f12aa4c22657935f3816bbbcc0894f30dc368f2